server/http_server.py

```
@@ -0,0 +1,493 @@
+            }
+        
+        return JSONResponse(content=response)
+        
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"Unexpected error in MCP endpoint: {e}")
+        raise HTTPException(status_code=500, detail="Internal server error")
+
+
+def run_server(host: str = "0.0.0.0", port: int = 8000):
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)